.storybook/main.js

```diff
@@ -6,21 +6,8 @@ const AppSourceDir = path.join(__dirname, '..', 'src');
 module.exports = {
     stories:
         process.env.NODE_ENV === 'production'
-            ? // Change below to ['../src/Components/**/*.stories.tsx'] to include Card stories
-              [
-                  '../src/Components/**/*.stories.tsx',
-                  '../src/Models/**/*.stories.tsx',
-                  '../src/Pages/**/*.stories.tsx'
-              ]
-            : // Change below to ['../src/**/*.stories.tsx', '../src/**/*.stories.local.tsx'] to include Card stories
-              [
-                  '../src/Components/**/*.stories.tsx',
-                  '../src/Models/**/*.stories.tsx',
-                  '../src/Pages/**/*.stories.tsx',
-                  '../src/Components/**/*.stories.local.tsx',
-                  '../src/Models/**/*.stories.local.tsx',
-                  '../src/Pages/**/*.stories.local.tsx'
-              ],
+            ? ['../src/Components/**/*.stories.tsx']
```

Comment:
so what is this change for and are we sure we do not want to export src/Pages stories in production?

Reply:
Good catch, I'll make the change before merging this PR. This change is reverting back to the old way stories were exported, since I had specified paths to ignore Card stories before.